config/M64.config

```diff
@@ -24,7 +24,7 @@ process {
             }
         }
     withName: call_gSV_Manta {
-        cpus = 1
+        cpus = 30
```

Comment:
**(question: non-blocking)** @Faizal-Eeman can you remind us of the optimal #CPU based on the existing benchmarking?

Reply:
Based on the paper you shared (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10976732/figure/Fig4/), it seems like higher the CPU better is the runtime for Manta. So I set 30 CPUs, so that we still have enough CPUs left for allot to Delly (& other callers) in the future.